src/pages/our-mission.en.tsx

```diff
@@ -8,9 +8,9 @@ import PageHero from "../components/page-hero";
 import { ReadMoreLink } from "../components/read-more";
 import { Accordion } from "../components/accordion";
 
-const ResponsiveSectionTitle: React.FC<{ children: React.ReactNode }> = ({
-  children,
-}) => (
+const ResponsiveSectionTitle: React.FC<{
+  children: React.ReactNode;
+}> = ({ children }) => (
```

Comment:
Hmmm I'm curious how this got reformatted in this PR? Doesn't seem to make a difference, just odd

Reply:
oops - I've got to double check all the changed files, that shouldn't have been in there. At one point I was going to use the component in the policy page so added `export` and that must have pushed it over the line length for prettier. Then when I later took out the export I didn't notice that the formatting had changed and didn't revert.